apps/sparo-lib/src/services/SparoProfileService.ts

```diff
@@ -98,7 +98,7 @@ export class SparoProfileService {
   }
 
   private static _getProfileName(profilePath: string): string {
-    const pathArr: string[] = profilePath.split('/');
+    const pathArr: string[] = profilePath.split(path.sep);
```

Comment:
It would be better to use parsing that is consistent on both platforms and supports paths with mixed slashes.

Reply:
Something like:
```ts
const parsed = path.parse('/home/user/dir/file.txt')
if (parsed.ext === '.json') {
  return parsed.name
}
```

Also I don't understand why this function does `return last;` -- that seems like a bug.  For example what if there is a `file common/sparo-profiles/README`.

Reply:
Nice catch. I will fix it in the next PR.

Reply:
Nice catch. I will fix it in the next PR.